### PR TITLE
Web3 challenge solutions

### PR DESCRIPTION
**Pull request by rishabhkeshan**
Added Web3 Challenge Solutions for the 3 Web3 challenges NFT Takeover, Mint the Honey Pot and Withdraw more ETH than deposited.

### CODE REVIEW

File: docs/modules/ROOT/pages/appendix/solutions.adoc [hunk 1]
```diff
@@ -3025,4 +3037,17 @@ solution should trigger accordingly.
 
 === Withdraw more ETH from the new wallet than you deposited
 
+. Visit the Digital Wallet from the Orders & Payment Section in the Account dropdown
```
Comment (ShubhamPalriwala) [inline docs/modules/ROOT/pages/appendix/solutions.adoc:3040]: Can you mention the endpoint here please?

**Review by ShubhamPalriwala — COMMENTED** [review]
Hey @rishabhkeshan, thanks for the documentation of the challenges. Great work, just some small nits:

- Either have full-stops everywhere or don't at all, refer to how we do it for other challenges
- Please outlink to whatever 3rd party services/apps you mention. Eg: Metamask, wiki for Reentrancy attack, Remix etc etc. This will allow an eager user to read more about it.

Rest all is pretty solid!

File: docs/modules/ROOT/pages/appendix/solutions.adoc (hunk 1 above)
Comment (rishabhkeshan) [reply docs/modules/ROOT/pages/appendix/solutions.adoc]: Hey @ShubhamPalriwala have added the necessary links and wikis to some good docs for the attacks mentioned, should be good to go.

**Review by ShubhamPalriwala — APPROVED** [review]
Thanks! Changes look good to me 🚀 

Just want to know, you tested them thoroughly in both exported HTML as well as PDF right, specifically, the images and the hyperlinks?

**Pull request merged**